manifests/postgresql_settings.pp

```diff
@@ -125,11 +124,12 @@
       db                    => 'pe-puppetdb',
       table_name            => 'reports',
       table_attribute       => 'autovacuum_vacuum_scale_factor',
-      table_attribute_value => "${reports_autovacuum_vacuum_scale_factor}",
+      table_attribute_value => String($reports_autovacuum_vacuum_scale_factor),
     }
   }
 
   if $manage_reports_autovacuum_cost_delay {
     pe_databases::set_puppetdb_table_autovacuum_cost_delay_zero { 'reports' : }
   }
 }
+
```

Comment:
I suspect it's like all whitespace rules - there to prevent pointless problems with diffs on merges.  It doesn't require an empty blank line, though - if the last line ends with a newline it should be ok, shouldn't it?